nlmod/dims/grid.py

```diff
@@ -840,6 +836,31 @@ def lcid_to_reclist(
     return reclist
 
 
+def cols_to_reclist(ds, cellids, *args, cellid_column=0):
```

Comment:
I think at some point this function should replace the `lrc_to_reclist` and the `lcid_to_reclist` functions because they something similar. I would not do it now but I can make an issue of this if you agree?

Reply:
I agree!